### PR TITLE
Inventory-like workloads filtering

### PR DESCRIPTION
**Pull request by semtexzv**
Implemented using new nested query parsing functionality

### CODE REVIEW

File: manager/controllers/systems_test.go [hunk 1]
```diff
@@ -136,6 +136,22 @@ func TestSystemsTags(t *testing.T) {
 	assert.Equal(t, "00000000-0000-0000-0000-000000000001", output.Data[0].ID)
 }
 
+func TestSystemsWorkloads(t *testing.T) {
```
Comment (Josca) [inline manager/controllers/systems_test.go:139]: I would prefer more tests here. What about some edge cases? Does it work for sap_system=false filter? Does it works for empty "sap_sids" input?

**Review by Josca — CHANGES_REQUESTED** [review]
just one thing, thanks

File: manager/controllers/systems_test.go (hunk 1 above)
Comment (semtexzv) [reply manager/controllers/systems_test.go:139]: Added

**Review by Josca — APPROVED** [review]
thanks
